src/main/scala/workshop3/assignments/solutions/FileIOSolution.scala

```diff
@@ -82,21 +82,21 @@ object FileIOSolution extends App {
     val products = readProducts(productInput)
     val customers = readCustomers(customerInput)
 
-    val r1 = report1(orders, products, customers)
-    val r2 = report2(orders, products, customers)
+    val wants = report1(orders, products, customers)
```

Comment:
maybe we should, but that would also involve changing the assignment itself. I prefer to wait with that as it might confuse with the workshop coming up tomorrow...